src/algorithms/calorimetry/CalorimeterClusterMerger.cc

```diff
@@ -104,8 +105,8 @@ void CalorimeterClusterMerger::AlgorithmProcess() {
         new_clus->setTime(time / energy);
         new_clus->setNhits(nhits);
         new_clus->setPosition(position / energy);
-        if (false) {
-          LOG_INFO(default_cout_logger) << "   --> Merged cluster with energy: " << new_clus->getEnergy() << LOG_END;
+        if (m_logger->level() <=spdlog::level::debug) {
```

Comment:
`If` could be removed

Reply:
leaving as close to the original as possible